core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -134,21 +134,21 @@
   public static final PropertyKey UNDERFS_S3_UPLOAD_THREADS_MAX =
       create(Name.UNDERFS_S3_UPLOAD_THREADS_MAX, 20);
   public static final PropertyKey UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS =
-      create(Name.UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS, 60000);
+      create(Name.UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS, "1min");
   public static final PropertyKey UNDERFS_S3A_DIRECTORY_SUFFIX =
       create(Name.UNDERFS_S3A_DIRECTORY_SUFFIX, "/");
   public static final PropertyKey UNDERFS_S3A_INHERIT_ACL =
       create(Name.UNDERFS_S3A_INHERIT_ACL, true);
   public static final PropertyKey UNDERFS_S3A_REQUEST_TIMEOUT =
-      create(Name.UNDERFS_S3A_REQUEST_TIMEOUT_MS, 60000);
```

Comment:
@calvinjia should the caller of `UNDERFS_S3A_REQUEST_TIMEOUT_MS` and `UNDERFS_S3A_SOCKET_TIMEOUT_MS` be updated as well?